wallet/src/actors/app/methods.rs

```diff
@@ -22,12 +23,12 @@ impl App {
     /// Return a new subscription id for a session.
     pub fn next_subscription_id(
         &mut self,
-        session_id: types::SessionId,
+        session_id: &types::SessionId,
     ) -> Result<types::SubscriptionId> {
-        if self.state.is_session_active(&session_id) {
+        if self.state.is_session_active(session_id) {
             // We are re-using the session id as the subscription id, this is because using a number
             // can let any client call the unsubscribe method for any other session.
```

Comment:
If we ever want to support more than one subscription id per session, we can just use `format!("{}-{}", session_id, subscription_id)` as the subscription id.

Reply:
The silly thing here is that a session already means one subscriber through one WS connection. I don't think we need more than one sink.

Although who knows, wasn't Bill Gates who said that 640KB was more memory than anyone will ever need? :rofl: